### ISSUE TITLE
A question about React menthod

### ISSUE THREAD

**Issue by jimo17** [deeplearning-wisc/react#7]
Hello. Thanks for open-sourcing the code. I am very interested in your paper. I have a question. Your code is a combination of ReAct and other OOD detection methods. If I want to simply use the ReAct method for OOD detection, can I directly use the logit value after activation truncated as the score?
`logits = forward_threshold(inputs, model)`

**Comment by sunyiyou** [comment]
Thanks for your interest in our work! 

A quick answer is yes, we show in the paper that ReACT is compatible with different scoring functions like ODIN, MSP, Energy, etc. Directly using max logit score should also work well. 

Note that Energy score is a smooth approximation to the max logit, you will see a similar performance.

**Comment by jimo17** [comment]
Thanks for your answer. Excuse me, does the ReAct method in Table 1 of the paper use the logit value after activation truncated as the score?

**Comment by sunyiyou** [comment]
More accurately. Say logit is f(x) = w h(x) + b. We report Energy score which is Logsumexp(f(x)), you can see it is an approximation to max f(x) (max over all class logits).

**Comment by YixuanLi** [comment]
Thanks for your interest in our work. Just to add some quick comments in the thread:

Compare to logit, the energy score has a clear mathematical interpretation from a likelihood perspective. Please see  the following papers for an explanation: 

https://arxiv.org/abs/2010.03759
https://arxiv.org/abs/2112.00787
